ssl/t1_trce.c

```diff
@@ -428,6 +428,10 @@ static ssl_trace_tbl ssl_ciphers_tbl[] = {
     {0xCCAD, "TLS_DHE_PSK_WITH_CHACHA20_POLY1305"},
     {0xCCAE, "TLS_RSA_PSK_WITH_CHACHA20_POLY1305"},
     {0x1301, "TLS_AES_128_GCM_SHA256"},
+    {0x1302, "TLS_AES_256_GCM_SHA384"},
```

Comment:
Well there are a couple of reasons. This table has the official (as defined in spec) names for the ciphersuites which don't always agree with our historical versions, where the two agree we could use the _TXT_ version. The cipher constants, yeah we could use those where they exist (this table also includes ciphersuites we don't support though for debugging purposes.

BTW that table was originally generated from a script which extracted everthing from the relevant specs.